aten/src/ATen/native/native_functions.yaml

```diff
@@ -280,6 +280,7 @@
     HammerBlade: add
     SparseCPU: add_sparse
     SparseCUDA: add_sparse
+    SparseHammerBlade: add_sparse
```

Comment:
@vb000 I think @zzy82518996 said he didn't actually implement those kernels, and these are what he intended to implement. What do you think? Should we not declare a kernel if it is not implemented yet? I kind of did the same thing ... but it was for the undergrads so they don't have to touch native_functions.yaml

Reply:
I see it's a nice way to figure out how call sequence works. A dummy kernel with an error message sounds good?

Reply:
Basically, anyway we could know where the error occurred. If not implementing anything would generate a reasonable error message, I guess that'd be fine as well!

Reply:
My concern is that, say we define `add_sparse` here for HammerBlade, and it calls xxx_stub eventually, then the error message is terrible. It tells you _some_ stub is not implemented for HammerBlade but does not say which and where it is called.

I would advocate for not defining for HammerBlade unless we are implementing it.